perllib/FixMyStreet/Script/ProcessUpdateFile.pm

```diff
@@ -61,13 +61,17 @@ sub process {
 
     die "Problem loading body\n" unless $self->body;
 
+    print "Dry run, not adding comments. Use --commit to add\n";
```

Comment:
Does this need to be displayed conditionally, based on the value of `$self->commit`?

Reply:
Oops ignore this, I wrote it before I saw the next commit!